rollupcreator/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16-bullseye-slim
+ARG NITRO_CONTRACTS_BRANCH=main
+RUN apt-get update && \
+    apt-get install -y git docker.io python3 build-essential curl jq
+WORKDIR /workspace  
+RUN git clone -b ${NITRO_CONTRACTS_BRANCH} https://github.com/OffchainLabs/nitro-contracts.git ./
```

Comment:
can we do regular clone and then checkout so that `NITRO_CONTRACTS_BRANCH` can be branch name, tag, or commit hash?

Reply:
Done